tools/visual-tester/README.md

```diff
@@ -144,7 +154,7 @@ This issue is not there when using `npm`.
 If you don't want to do this you could use `node`
 
 ```bash
-node ./dist/runner/cli.js --specs="./specs/**/*.vspec.js"
+./node_modules/.bin/tsc && node ./dist/runner/cli.js --specs="./specs/**/*.vspec.js"
```

Comment:
nit: Can we use `npx` here, or we just don't want to take chances about the global npm version?